### ISSUE TITLE
BUG: Drop table index on the UI does not work due to incorrect SQL generated

### ISSUE THREAD

**Issue by basonjui** [beekeeper-studio/beekeeper-studio#2290]
**Describe the bug**
I have not tested on **public** schema, but on a named schema of PostgreSQL, dropping the index using the UI will encounter the error `index "{schema_name}.{index_name}" does not exist`.

<img width="1132" alt="Screenshot 2024-06-27 at 16 47 43" src="https://github.com/beekeeper-studio/beekeeper-studio/assets/60636087/0128d14a-140b-49fc-a7cd-46f8b5a9ab31">

When I click the **Copy to SQL** button, it generates the SQL used to drop the index and I can clearly see why the error occurred.

SQL generated by Beekeeper Studio
```sql
DROP INDEX
  "{schema_name}.{index_name}"
```

Correct SQL to drop an index (in a specific schema)
```sql
DROP INDEX
  {schema_name}."{index_name}"
```

**To Reproduce**

1. Create the table using this DDL
```sql
CREATE SCHEMA IF NOT EXISTS ihs;

CREATE TABLE IF NOT EXISTS
  ihs.ship_status (
    id serial NOT NULL,
    status_code character(1) NOT NULL,
    status text NOT NULL,
    definitions text NULL,
    last_modified timestamp without time zone NOT NULL DEFAULT now()
  );

ALTER TABLE
  ihs.ship_status
ADD
  CONSTRAINT ship_status_pkey PRIMARY KEY (id);
  
CREATE UNIQUE INDEX IF NOT EXISTS ship_status_statuscode_key ON ihs.ship_status USING btree (status_code);
```
2. On the UI, right click the table `ihs.ship_status`, select `View Structure`, go to tab `Indexes`
3. Click button X on the index Name `ihs.ship_status_statuscode_key`
4. Click Apply button
5. See error

**Expected behavior**
See Error message:
```
There was a problem
index "ihs.ship_status_statuscode_key" does not exist
```

Click the down arrow next to Apply button, click `Copy to SQL`, and see the following SQL generated
```sql
DROP INDEX
  "ihs.ship_status_statuscode_key"
```

**Screenshots / Gifs**
<img width="1077" alt="Screenshot 2024-06-27 at 16 41 35" src="https://github.com/beekeeper-studio/beekeeper-studio/assets/60636087/21ace9dc-f686-4a32-976c-d7c8698ada09">
<img width="338" alt="Screenshot 2024-06-27 at 16 57 15" src="https://github.com/beekeeper-studio/beekeeper-studio/assets/60636087/fbca6e7f-3069-4848-bac0-a2749c71ab8e">

**Version Information (please complete the following information):**
 - OS: macOS Sonoma (Macbook Pro, 14-inch, 2023)
 - App Version: Beekeeper Studio Ultimate, version 4.6.1
 - Database type and version: PostgreSQL 12.19 on x86_64-pc-linux-gnu, compiled by gcc (GCC) 13.2.1 20240316 (Red Hat 13.2.1-7), 64-bit

**Additional context**
When fixed the SQL generated by Beekeeper (mentioned above in the correct SQL section), the index was dropped successfully.
<img width="498" alt="Screenshot 2024-06-27 at 17 16 36" src="https://github.com/beekeeper-studio/beekeeper-studio/assets/60636087/bdd487e8-8ea6-4a60-8368-0a2d5f10f7eb">

**Comment by rathboma** [comment]
Hey!

Thanks for the report. We'll get this fixed and add a unit test also.

**Comment by rathboma** [comment]
I think we just messed up the identifier escaping when we refactored our PSQL code. Should be an easy fix.

Big big thanks for the super detailed report.

**Comment by basonjui** [comment]
No worries, thanks for making Beekeeper Studio so awesome. Found out about it during my first job (on a Ubuntu) 2+ years ago and haven't stopped using it since then! :) 

Was extremely glad to find out that you guys also support macOS.

**Comment by azmy60** [comment]
Fixed by #2292
